base/exports.jl

```diff
@@ -1430,6 +1430,7 @@ export
     @inbounds,
     @fastmath,
     @simd,
+    @shareindexes,
```

Comment:
Well, I guess you could say the iterator is being shared too. So I think we could go with that name.